### DISCUSSION TITLE
✨ Stash

### DISCUSSION THREAD

**Discussion by JigSawFr** [runtipi/runtipi-appstore#2209] [Requests]
### Description

Stash is a self-hosted webapp written in Go which organizes and serves your porn.
demo image

Stash gathers information about videos in your collection from the internet, and is extensible through the use of community-built plugins for a large number of content producers and sites.
Stash supports a wide variety of both video and image formats.
You can tag videos and find them later.
Stash provides statistics about performers, tags, studios and more.

### Official Website

https://stashapp.cc/

### Documentation link

https://docs.stashapp.cc/

### Last application release & date

0.24.3 (15/01/2024)

### Application license

AGPL-3.0 license

### Source code repository link

https://github.com/stashapp/stash

### Docker image link

https://github.com/stashapp/stash/blob/develop/docker/production/docker-compose.yml

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote 👍 this discussion to show your interest in this request! 

Thanks ⛺

**Comment by JigSawFr** [comment]


**Reply by steveiliop56** [reply]
No it was not

**Reply by JigSawFr** [reply]
My little pig 💯
